pkg/insights/insightsclient/requests.go

```diff
@@ -42,9 +43,9 @@ func (c *Client) SendAndGetID(ctx context.Context, endpoint string, source Sourc
 	resp, err := c.client.Do(req)
 	if err != nil {
 		klog.V(4).Infof("Unable to build a request, possible invalid token: %v", err)
-		// if the request is not build, for example because of invalid endpoint,(maybe some problem with DNS), we want to have recorded about it in metrics as well.
-		counterRequestSend.WithLabelValues(c.metricsName, "0").Inc()
-		return "", fmt.Errorf("unable to build request to connect to Insights server: %v", err)
+		// if the request is not build, for example because of invalid endpoint,(maybe some problem with DNS), we want to have record about it in metrics as well.
+		insights.IncrementCounterRequestSend("0")
```

Comment:
Send `defaultCode` (variable or const for default 0 value) as it is, the data transformation should take place inside `IncrementCounterRequestSend` function

Reply:
e.g. insights.IncrementCounterRequestSend(defaultCode)

I think it's more clear what the 0 means, without even reading the comment above.